torch/distributed/utils.py

```diff
@@ -0,0 +1,65 @@
+import torch
+
+def _parse_remote_device(remote_device: str):
```

Comment:
I understand that compared with `_parse_remote_device` in `torch/distributed/rpc/utils.py`, here we also want to support another format `rank:<rank>/<device>`. Shall we unify these two very similar functions?